client/lib/directly/index.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+import { noop } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+// import { loadScript } from 'lib/load-script';
+
+const DIRECTLY_SCRIPT_PATH = 'https://www.directly.com/widgets/rtm/embed.js';
+const DEFAULT_RTM_WIDGET_OPTIONS = {
```

Comment:
> is that common for this kind of thing in Calypso?

It was a necessity for olark since not all users shared the same configuration. My thoughts are that we might have the same issue here where we would want to offer a different configuration for `pt-br` or `es` locale users than we would want to for `en` users and we could use the wp.com api/filters etc to make those business decisions. Either way the config doesn't really belong here. If you think sticking it in the API would be pointless and overkill then the next best place would be in the config directory starting with [config/development.json](https://github.com/Automattic/wp-calypso/blob/master/config/development.json)